### PR TITLE
Update configure-endpoints-vdi.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated downlevel SKUs

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6005

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-endpoints-vdi.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ The following steps will guide you through onboarding VDI devices and will highl
 7. Use the search function by entering the device name and select **Device** as search type.
 
 
-## For downlevel SKUs
+## For downlevel SKUs (Windows Server 2008R2/2012R2/2016)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-vdi.md]:
```suggestion
## For downlevel SKUs (Windows Server 2008 R2/2012 R2/2016)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba Please copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**